tools/ci_build/github/azure-pipelines/android-x86_64-crosscompile-ci-pipeline.yml

```diff
@@ -209,7 +208,6 @@ stages:
 
       - script: |
           python3 tools/ci_build/build.py \
-          --enable_lto \
```

Comment:
Why is this line removed?

Reply:
I think this was removed by mistake, or it was causing some built error. I don't remember why; it has been weeks.  I can try reenable it later and see if it still causes trouble's